src/main/java/com/cascadebot/cascadebot/commandmeta/CommandContext.java

```diff
@@ -248,6 +250,42 @@ public MessageAction replyImage(String url) {
         }
     }
 
+    public void replyUsage(ICommandExecutable command) {
```

Comment:
Would this be better to have a `String` as a parameter? This would allow to get a usage easily from elsewhere other than in the command.

Reply:
We could add a string method, but we should still have reply usage as a fast way to send it

Reply:
I don't see the need